lib/internal/repl.js

```diff
@@ -167,6 +168,12 @@ function setupHistory(repl, historyPath, oldHistoryPath, ready) {
                             'REPL session history will not be persisted.\n');
       }
       if (!threw) {
+        deprecate(
```

Comment:
It doesn't seem correct to me. Who calls the function returned by `deprecate()`?

Reply:
@lpinca sorry, I didn't read internal/util source correctly, I thought deprecate auto-printed the deprecation. updated my pr.